tests/TestLeibnitz.elm

```diff
@@ -0,0 +1,105 @@
+module TestLeibnitz exposing (..)
+
+import Expect exposing (Expectation)
+import Formula exposing (Formula(..))
+import Formula.Parser
+import Formula.Signed exposing (Signed(..))
+import Term exposing (Term(..))
+import Tableau exposing (..)
+import Test exposing (..)
+import Validate
+import Validation.Common
+import Validation.Rules.Leibnitz
+import Zipper exposing (..)
+
+
+tableauExample =
+    { node =
+        { id = 1
+        , value = ""
+        , references = [{ str = "1", up = Just 0 }]
+        , formula = Formula.Parser.parseSigned ""
+        , gui = defGUI
+        }
+    , ext = Open
+    }
+
+
+zipperExample =
+    ( tableauExample, [] )
```

Comment:
You should prefer using the interface functions over implementation details of data structures:
```elm
zipperExample =
    zipper tableauExample
```